python-lib/nlp_utils.py

```diff
@@ -40,6 +40,22 @@ def get_span_text(span: Span, lemmatize: bool) -> AnyStr:
     return span.lemma_ if lemmatize else span.text
 
 
-def unicode_normalize_text(texts: List[AnyStr]) -> List[AnyStr]:
-    """Return a list of texts NFD normalized"""
-    return [unicodedata.normalize("NFD", text) for text in texts]
+def unicode_normalize_text(
+    text: AnyStr, use_nfc: bool = False, normalize_diacritics: bool = False
+):
+    """Apply unicode_normalization to text
+
+    Args:
+        text (str): Text to normalize with NFD or NFC norm.
+        use_nfc (bool): Apply NFC norm if True, NFD otherwise.
+        normalize_diacritics(bool): if True, remove diacritics after unicode normalizing.
+
+    Returns:
+        str: Text normalized with NFC or NFD norm.
+
+    """
+    norm = "NFC" if use_nfc else "NFD"
```

Comment:
Here would be a good place to use an Enum :) It allows you to define only the available values and reduce to the maximum the problem of string typing.
I don't know if you have a constants file, but you could define there :
```python
from enum import Enum

class UNICODE_NORMALIZATION(Enum):
    NFC = "NFC"
    NFD = "NFD"
```